odoo/addons/base/models/ir_rule.py

```diff
@@ -221,6 +221,9 @@ def _make_access_error(self, operation, records):
 
         resolution_info = _("If you really, really need access, perhaps you can win over your friendly administrator with a batch of freshly baked cookies.")
 
+        import traceback
```

Comment:
This has nothing to do with this PR. Please made a separate one.
Also, there are multiple functions generating errors. Why only change this one?
The traceback is already in the exception, if you need printing it, it is surely not to the end-users.

Reply:
@kmagusiak it’s to debug a test that only fail on the runbot
The task is still in dev column in the project, not yet ready to be reviewed